server/v2/stf/cache.go

```diff
@@ -0,0 +1,65 @@
+package stf
+
+import (
+	// "fmt"
+	"unsafe"
+)
+
+// Container is a map that stores objects in any
+type Container struct {
+	m map[string]any
```

Comment:
are we assuming a string is unique? there will not be conflicts?

Reply:
Yes, currently its only using for item, the key is item.prefix

Reply:
can we do a simple wrapper for a mock collection item? would be nice to keep the dependency graph clean